endpoints/endpoints.md

```diff
@@ -12,75 +14,81 @@ mock_responses:
+ "profile_photo":"https://dummyimage.com/161x100.png"}
+ ```
+
+- `PUT` method: parameters: `(id,username,password,email,display_name,profile_photo)` response: `{[updated:true/false]}` description: updates user details
+
+mock_response:
 ```json
-{
-   "valid":"false",
-   "id":0
-}
+{"updated":true,}
 ```
 
-3. HTTP: `/create_task` : method: `POST` parameters: `(id,user_id,title,description,due_date,status)` response: `{[created:true/false],[id:]}` description: returns true if task has been created 
+- `DELETE` method: parameters: `(id)` response: `{[deleed:true/false]}` description: deletes user
 
 mock_response:
 ```json
-{"created":true,}
+{"deleted":true,}
 ```
 
-4. HTTP: `/tasks` : method: `GET` parameters: `(user_id)` response: `{for task in tasks:{id:,title:,description:,due_date:,status:}` description: returns all tasks for the user ID
+
+2. HTTP: `/tasks`
+
+- `GET` method: parameters: `(id)` response: `{for task in tasks:{id:,title:,description:,due_date:,status:}` description: returns all tasks for the user ID
 
 mock_response:
 ```json
 [{"id":1,"title":"Pick Trash","description":"remember to pick trash in the backyard","due_date":"7/14/2021","status":"created"},
 {"id":2,"title":"Go to hospital","description":"Been feeling funny, need to go to doctor's appointment","due_date":"3/20/2021","status":"done"},
 {"id":3,"title":"Apply for new role","description":"Apply for all the bookmarked roles I did last week","due_date":"4/23/2021","status":"progress"}]
 ```
-5. HTTP: `/update_task` : method: `PUT` parameters: `(id,title,description,due_date)` response: `{[updated:true/false]}` description: updates task details
 
-mock_response:
-```json
-{"updated":true,}
-```
-6. HTTP: `/delete_user` : method: `DELETE` parameters: `(id)` response: `{[deleed:true/false]}` description: deletes user
+- `POST`  : method: parameters: `(id,user_id,title,description,due_date,status)` response: `{[created:true/false],[id:]}` description: returns true if task has been created 
 
 mock_response:
 ```json
-{"deleted":true,}
+{"created":true,}
 ```
 
-7. HTTP: `/user` : method: `GET` parameters: `(id)` response: `{username:,email:,display_name:,profile_photo}` description: returns user details for an already authenticated user
-
-mock_response:
-```json
-{"username":"ernnjf",
- "email":"x.y@gmail.com",
- "display_name":"Abc Def",
- "profile_photo":"https://dummyimage.com/161x100.png"}
- ```
- 
-8. HTTP: `/update_user` : method: `PUT` parameters: `(id,username,password,email,display_name)` response: `{[updated:true/false]}` description: updates user details
+- `PUT` method: parameters: `(id,title,description,due_date)` response: `{[updated:true/false]}` description: updates task details
 
 mock_response:
 ```json
 {"updated":true,}
 ```
 
-9. HTTP: `/delete_task` : method: `DELETE` parameters: `(id)` response: `{[deleted:true/false]}` description: deletes task
+- `DELETE` method: parameters: `(id)` response: `{[deleted:true/false]}` description: deletes task
```

Comment:
@iannjari  we can return a string "task deleted successfully"

Reply:
Okay, lemme implement this then merge